python/dgl/nn/pytorch/gt/degree_encoder.py

```diff
@@ -31,10 +29,17 @@ class DegreeEncoder(nn.Module):
     -------
     >>> import dgl
     >>> from dgl.nn import DegreeEncoder
+    >>> import torch as th
+    >>> from torch.nn.utils.rnn import pad_sequence
 
-    >>> g = dgl.graph(([0,0,0,1,1,2,3,3], [1,2,3,0,3,0,0,1]))
+    >>> g1 = dgl.graph(([0,0,0,1,1,2,3,3], [1,2,3,0,3,0,0,1]))
+    >>> g2 = dgl.graph(([0,1], [1,0]))
+    >>> in_degree = pad_sequence([g1.in_degrees(), g2.in_degrees()], batch_first=True)
+    >>> out_degree = pad_sequence([g1.out_degrees(), g2.out_degrees()], batch_first=True)
     >>> degree_encoder = DegreeEncoder(5, 16)
-    >>> degree_embedding = degree_encoder(g)
+    >>> degree_embedding = degree_encoder(th.stack((in_degree, out_degree)))
```

Comment:
maybe print the input of `degree_encoder` first for those who are not familiar with `pad_sequence`